front/app/containers/ProjectsShowPage/shared/events/index.tsx

```diff
@@ -29,16 +43,49 @@ const StyledEventCard = styled(EventCard)`
   margin-bottom: 30px;
 `;
 
-interface Props {
+interface InputProps {
   projectId: string;
   className?: string;
 }
 
-const EventsContainer = memo<Props>(({ projectId, className }) => {
-  const project = useProject({ projectId });
+interface DataProps {
+  ideasLoaded: boolean;
+}
+
+interface Props extends InputProps, DataProps {}
+
+const allHaveLoaded = (...args) => args.every((arg) => !isNilOrError(arg));
+
+const EventsContainer = memo<Props>(({ projectId, className, ideasLoaded }) => {
   const { events } = useEvents({ projectIds: [projectId] });
+  const locale = useLocale();
+  const tenant = useAppConfiguration();
+  const phases = usePhases(projectId);
+
+  useEffect(() => {
+    const scrollToEventId = getScrollToEventId();
 
-  if (!isNilOrError(project) && !isNilOrError(events) && events.length > 0) {
+    if (
+      scrollToEventId !== null &&
+      ideasLoaded &&
+      allHaveLoaded(events, locale, tenant, phases)
```

Comment:
I think what you're doing here is making sure things are loaded so the page is already the right height before you scroll, if I'm mistaken I could be entering a very long useless argument, so... let me know haha. But I'm worried about maintainability. You need a way to communicate that the scroll to event functionality relies on the project page being the right size once a fixed list of resources are loaded to any person who could need that info, like a dev tasked to add a component to the project page that'd substantially change the height of the project page and rely on data fetching without breaking this. With the expected move towards an open modular system, I think it's important to make sure this kind of assumptions are well communicated and maintained. 
Tl;dr : I think we should document this and once the project page is extensible make sure this list is too, and for now add a good enough test to make sure no-one breaks this.